src/main/java/de/caritas/cob/messageservice/api/service/helper/ServiceHelper.java

```diff
@@ -9,6 +11,7 @@
 import de.caritas.cob.messageservice.api.helper.AuthenticatedUser;
 
 @Component
+@Slf4j
 public class ServiceHelper {
```

Comment:
please refactor this class to add it more meaningful name like HttpHeadersProvider, also Junit

Reply:
this was an existing class